controllers/loadtest_controller.go

```diff
@@ -139,12 +155,38 @@ func (r *LoadTestReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 		}
 	}
 
+	reQueueTime := getRequeueTime(test, previousStatus, log)
```

Comment:
nit: I would keep consistent case with function and make the variable `requeueTime`.